test/env_test.rb

```diff
@@ -72,11 +72,6 @@ def test_request_create_stores_ssl_options
     assert_equal false, env.ssl.verify
   end
 
-  def test_request_create_stores_proxy_options
-    env = make_env
-    assert_equal 'proxy.com', env.request.proxy.host
-  end
```

Comment:
Makes sense then.
I'd assume the coverage of this test have been moved to the new tests so happy to stay without it 👍 
Thanks for the clarification 😄